shared/utils/processElements.ts

```diff
@@ -0,0 +1,24 @@
+import {
+    ContentItem,
+    FieldModels,
+    Fields,
+} from 'kentico-cloud-delivery';
+
+export const processLinkedItemsElement = (linkedItemsElement: ContentItem[]): string[] =>
+    linkedItemsElement.map((contentItem: ContentItem) => contentItem.system.codename);
```

Comment:
Just to be sure it is mentioned, please, remove hard-coded types from selectors - types in selectors should be inferred, otherwise, TS does not type-check selectors properly...